playwright/_impl/_page.py

```diff
@@ -83,6 +83,7 @@
     serialize_argument,
 )
 from playwright._impl._network import Request, Response, Route, serialize_headers
+from playwright._impl._set_input_files_helpers import convert_input_files
```

Comment:
looks unused

Reply:
yes, will be removed when I lint everything, I hope 😄